src/Components/EnvironmentPicker/EnvironmentPickerManager.ts

```diff
@@ -13,10 +13,15 @@ import {
 export const getEnvironmentDisplayText = (env: string | IAzureResource) => {
```

Comment:
![image](https://user-images.githubusercontent.com/57726991/186237450-35b45b69-d1a0-4556-a6a2-352587e4554f.png)

Reply:
had commented and marked as "resolved" this on the original PR